.travis.yml

```diff
@@ -1,54 +1,21 @@
 language: node_js
 node_js:
-  - '12'
-  - '10'
-  - '8'
```

Comment:
why aren't we testing on 8 if 8 is in our `engines`?

Reply:
We do.

But instead of ignoring, we only include the only test case for Node 8

Reply:
aha! It's been so long since I used travis I've forgotten how the config works 😅